mcstatus/__main__.py

```diff
@@ -1,75 +1,155 @@
 from __future__ import annotations
```

Comment:
Just though, that would be great to create a page for CLI in docs. Though feel free to not, it's not a requirement (`mcstatus --help` is still pretty much enough)

Reply:
I've updated the README to line up with the new features. I'll leave a proper documentation page to you.

Reply:
To be honest, it is better to revert that change. We already had `mcstatus --help` in our README, and it was always outdated (that's why we removed it)

Reply:
Do you feel strongly about this? Code review should prevent outdated documentation.

Reply:
> Do you feel strongly about this?

Kind of, it's a mistake we already made

> code review should prevent outdated documentation

Unfortunately, without automatized tools it is hard to remember this. That's why it is a bad idea to keep `--help` in README: one or two changes to CLI and it is no longer helpful, but confusing.

I would agree on `mcstatus --help` which runs on build time in docs, though you don't have to do this, as I don't even know if it is possible

Reply:
I really don't think it's as dangerous as you say, but I have made the change.

I agree test cases that run the CLI would be very good, maybe even to check the actual functionality.

Reply:
Or, add a comment in the CLI code telling the editor to update the readme. It's not an impossible problem :(